astroid/protocols.py

```diff
@@ -129,6 +107,8 @@ def const_infer_binary_op(
     context: InferenceContext,
     _: SuccessfulInferenceResult,
 ) -> Generator[ConstFactoryResult | util.UninferableBase, None, None]:
+    from astroid.nodes import const_factory  # pylint: disable=import-outside-toplevel
```

Comment:
On line 57 we already use this from `nodes` why did you change this?

Reply:
Good catch. I shuffled this method out of this file and back into it, and missed this diff when I brought it back. Thank you!